superset-frontend/src/features/home/RightMenu.tsx

```diff
@@ -500,15 +494,6 @@ const RightMenu = ({
             <ThemeEditor theme={themeEditorTheme} setTheme={setTheme} />
           </span>
         )}
-        {isFeatureEnabled(FeatureFlag.ThemeEnableDarkThemeSwitch) && (
-          <span>
-            <ThemeSelect
```

Comment:
@mistercrunch isn't this the component that controls the option for the user for default / dark / system settings mode? I don't think this is a leftover. This was meant to be here.